package.json

```diff
@@ -5,7 +5,7 @@
         "less" : ">=1.5.0",
         "gulp": ">=3.8.0",
         "gulp-less": ">=1.3.0",
-        "gulp-minify-css": ">=0.3.0"
+        "gulp-clean-css": ">=2.3.0"
```

Comment:
I know the previous code did this, but `>=` is a very bad idea, especially without any shrinkwrap file. `^2.3.0` is safer, since it doesn't match 3.x.